pkg/storkctl/clusterpair.go

```diff
@@ -249,5 +276,8 @@ func newGenerateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptio
 		},
 	}
 
+	generateClusterPairCommand.Flags().StringVarP(&storageIP, "storage-ip", "", "", "IP of one of storage node")
```

Comment:
These are Portwrox specific, and shouldn't be explicit here. I think we should allow this to be passed in as
```
--options=ip=10.0.0.0,port=9001,token=abcde
```
In the future if drivers require more options we wouldn't need to update storkctl